data/json/overmap/overmap_mutable/river_cave.json

```
@@ -0,0 +1,69 @@
+[
+  {
+    "type": "overmap_special",
+    "id": "river_cave",
+    "subtype": "mutable",
+    "locations": [ "subterranean_empty" ],
```

```suggestion
    "locations": [ "subterranean_empty", "land" ],
```
The tunnel_to_surface joint specifies going into the "land" location, but you havent set it as valid here

---

Hmm still spawns correctly when it lands on a river tile once in a blue moon but mostly spawns in the middle of nowhere with a forest swamp above it that lacks the ramp down to enter the cave.